.github/actions/deploy/deploy.sh

```diff
@@ -76,7 +76,7 @@ EOF
 
 ssh -l $SSH_USERNAME -T $SSH_HOST <<EOF
   cd $SSH_DIRECTORY
-  php artisan down
+  php artisan down --render=maintenance
```

Comment:
Wouldn't it make more sense to call the file `update.blade.php`? It's not a regular maintenance, is it?

Reply:
Thanks, fixed.